lib/use_cases/administrator/create_signup_whitelist.rb

```diff
@@ -6,7 +6,7 @@ class CreateSignupWhitelist
       def execute(domains)
         return NOOP_REGEX if domains.empty?
 
-        '^.*@' + literal_dot(domains_list(domains) + '$')
+        '^.*@([a-zA-Z0-9.-]+)?' + literal_dot(domains_list(domains) + '$')
```

Comment:
this allows more than just subdomains, as the `.` is now optional.
`someone@maliciousgov.uk` will match

Reply:
suggest changing to `([a-zA-Z0-9-]+\.)?`